quizizz.py

```
@@ -0,0 +1,41 @@
+def check_room():
+    while True:
+        pin = str(random.randint(111111, 999999))
+        url = "https://game.quizizz.com/play-api/v5/checkRoom"
+
+        data = {
+            "roomCode": pin
+        }
+
+        response = requests.post(url, json=data).json()
+        if "success" in response:
+            responseCode = response['success']
```

Ruff (0.8.2)

Local variable `responseCode` is assigned to but never used

Remove assignment to unused variable `responseCode`

(F841)